tools/downloader/downloader.py

```diff
@@ -225,6 +235,11 @@ def main():
             output.mkdir(parents=True, exist_ok=True)
 
             for model_file in model.files:
+                # Download only files with specific precision.
+                prefix = str(model_file.name)[:str(model_file.name).find('/')]
+                if prefix in common.KNOWN_PRECISIONS and not prefix in requested_precisions:
+                    continue
```

Comment:
The precisions option for the downloader is clearly a separate feature, so I'm going to request that you submit it as a separate PR. You'll also need to document this option in the README.

Reply:
@IRDonch, sure, thank you!